pom.xml

```diff
@@ -101,15 +90,29 @@
     </testResources>
     <plugins>
       <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5</version>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <executions>
+              <execution>
+                  <id>display-info</id>
+                  <configuration>
+                      <rules>
+                          <enforceBytecodeVersion>
+                              <excludes combine.children="append">
+                                  <exclude>org.connectbot.jbcrypt</exclude>
+                              </excludes>
+                          </enforceBytecodeVersion>
+                      </rules>
+                  </configuration>
+              </execution>
+          </executions>
       </plugin>
     </plugins>
   </build>
 
   <properties>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <revision>build-217-jenkins-25</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <java.level>8</java.level>
```

Comment:
Still missing the githubRepo property, does the incrementalify command not work for you to automatically generate this?

Reply:
I cannot test it against the official artifactory, I do not have permissions, The trilead-api-plugin has the same configuration (no githubRepo var replacements) and works https://github.com/jenkinsci/trilead-api-plugin/blob/master/pom.xml e.g. https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/plugins/trilead-api/1.0.12-rc42.03cd13b7a5ae/

Reply:
>Also replace jenkinsci/your-repo occurrences in the scm section with a POM property named gitHubRepo. This will ensure that deployments from forked pull requests will give correct checkout information, used for example by plugin-compat-tester.

ok, this is for forks